src/runner/config/defaultconfig.go

```diff
@@ -15,3 +22,23 @@ func init() {
 
 	DefaultConfig = string(decoded)
 }
+
+func GetDefaultConfigURL() string {
+	isVersionEmbeddedToBinary, err := ota.IsVersionEmbeddedToBinary()
+	if err != nil || !isVersionEmbeddedToBinary {
+		return FallbackConfigURL
+	}
+
+	major, minor, patch, err := ota.ResolveVersion()
+	if err != nil {
+		return FallbackConfigURL
+	}
+
+	return buildDefaultConfigURLForVersion(major, minor, patch)
```

Comment:
ok, I see which use-case is being solved here. It would probably be wiser to use filters on jobs which I've added couple days ago instead of having two separate configs. Let me work on that

Reply:
It's completely up to you, @Arriven. Probably, this dynamic URL build is overkill for this